src/config/schema.rs

```diff
@@ -1,17 +1,22 @@
-use serde::{Deserialize, Serialize};
+use vector_config::configurable_component;
 use vector_core::config::LogNamespace;
 
 pub(crate) use crate::schema::Definition;
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq, Copy, Clone)]
+/// Schema options.
+#[configurable_component]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[serde(default, deny_unknown_fields)]
 pub struct Options {
+    /// Whether or not schema validation is enabled.
```

Comment:
```suggestion
    /// Whether or not schema is enabled.
```

Reply:
That doesn't really read right to me. What is "schema"?

I could see it being "schemas are enable"... but maybe I'm not fully grokking what this enables. Why would we "enable" schemas but not validate them?

Reply:
Same, perhaps @fuchsnj or @JeanMertz need to weigh in as I'm not super familiar with this part of the code. I'm not sure what the difference is between `enabled` and `validation` here.